### ISSUE TITLE
Dead or alive?

### ISSUE THREAD

**Issue by strubem** [gciruelos/musthe#10]
Are you still working on this?

**Comment by gciruelos** [comment]
Nope, it was an idea I had, it was never meant as a long term project.

If you want to contribute, fork, whatever, you are free to do it.

**Comment by strubem** [comment]
I have completely rewritten it, so forking and stuff would probably not make too much sense. But, still, lots of thanks: Before I had seen your code, I was always just trying to add semitones to root notes to get chords and scales and just couldn't figure out how to find out whether a note is a sharp or a flat.

**Comment by DanielOaks** [comment]
Just to be curious, are you going to be throwing up your library, @strubem ? (If you've rewritten it as a library).

I'd be interested in taking a look if you have.

**Comment by gciruelos** [comment]
+1 @DanielOaks 

thanks @strubem 
However I think the code is fairly ugly, so I'd like to see what you have done.

**Comment by strubem** [comment]
I guess I will, but I still have to write docstrings and tests. Also I am interested in learning all the other stuff before setting up a project. (Git, Sphinx, Pip, etc.)

But I have really done a few nice, working things: http://hastebin.com/humijulore.py

**Comment by DanielOaks** [comment]
Fair enough. There are some decentish guides around for how to do most of that other stuff. With the Sphinx documentation I'd look into hosting on Read The Docs if you aren't already, they're very useful.

If you do run into any issues or questions with those other things, feel free to email me. It can be a bit to digest all at once, especially with some resources being less-than-stellar for someone who hasn't done it before. :)
